Source/WebKit/WebProcess/Plugins/PDF/PDFPluginBase.h

```diff
@@ -256,6 +270,13 @@ class PDFPluginBase : public ThreadSafeRefCounted<PDFPluginBase>, public WebCore
     bool m_documentFinishedLoading { false };
     bool m_isBeingDestroyed { false };
     bool m_hasBeenDestroyed { false };
+
+#if PLATFORM(MAC)
+    RefPtr<PDFPluginAnnotation> m_activeAnnotation;
+#endif
```

Comment:
What's this platform guard about? Also, on that note, should we not make the `PDFPlugin*Annotation` sources be guarded by `ENABLE(PDF_PLUGIN)` instead of `ENABLE(LEGACY_PDF_PLUGIN)`?

Reply:
Well, I guess that second part can happen in a separate patch, if we keep this patch strictly for decoupling annotations from PDFLayerController. Though I reckon things will still build if you change the guards?

Reply:
The reason I added it was actually because of the `LEGACY_PDF_PLUGIN` guard which is also platform specific I believe. I think the reason this guard is in place is because of the use of `PDFAnnotation*Widget`s in some of the annotation types. I think a little bit of additional work on the `PDFPlugin*Annotation`s would be needed in order to get rid of the platform guard